community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/planDescription/PlanDescriptionArgumentSerializer.scala

```diff
@@ -60,6 +60,14 @@ object PlanDescriptionArgumentSerializer {
         val types = typeNames.mkString(":", "|:", "")
         val relInfo = if (!varLength && typeNames.isEmpty && rel.unnamed) "" else s"[$rel$types$asterisk]"
         s"($from)$left$relInfo$right($to)"
+      case CountNodesExpression(ident, label) =>
+        val node = label.map(l => ":" + l.name).mkString
+        s"count( ($node) )" + (if (ident.startsWith(" ")) "" else s" AS $ident")
+      case CountRelationshipsExpression(ident, startLabel, typeNames, endLabel) =>
+        val start = startLabel.map(l => ":" + l.name).mkString
+        val end = endLabel.map(l => ":" + l.name).mkString
+        val types = typeNames.map(t => t.name).mkString(":", "|:", "")
+        s"count( ($start)-[$types]-($end) )" + (if (ident.startsWith(" ")) "" else s" AS $ident")
```

Comment:
There are functions `notNamed` and `isNamed` in `PrefixNameGenerator` that could be used instead of `startsWith(" ")`